scripts/iql-lesson-sync/pyproject.toml

```
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "iql-lesson-sync"
+version = "0.0.1"
+authors = [
+  { name="Frank Harkins", email="frankharkins@hotmail.co.uk" },
+]
+description = "Tool for IBM Quantum Learning content creators to upload lessons through the API."
+readme = "README.md"
+requires-python = ">=3.7"
```

Py37 is EOL, so might as well to 3.8+

```suggestion
requires-python = ">=3.8"
```